### ISSUE TITLE
Does this currently work?

### ISSUE THREAD

**Issue by dnakdnak** [OptoCloud/VRC-OSC-ParticlePrinter#2]
I want to know how to use

**Comment by OptoCloud** [comment]
Yes, this works, I just never imagined it would get this popular, and I've been avoiding answering issues/PR's.
Might do a tiny bit of a rework soon

**Comment by OptoCloud** [comment]
I overlooked the broken comment from PR #3 which broke the syntax of the script, fixed it now lol
